Sources/Camille/Configs/Configs+CrossPostService.swift

```diff
@@ -1,7 +1,7 @@
 import Sugar
 
 extension Configs {
-    static let CrossPostService = CrossPostServiceConfig(
+    static let CrossPost = CrossPostServiceConfig(
```

Comment:
I haven't been super in-touch with the Swift 3 changes, and I realize the previous property was this as well, but it seems a little odd to me that these static properties are UpperCamelCased.